less/button.less

```diff
@@ -122,60 +129,62 @@ span.menuButton {
     margin-right: 0px;
     margin-left: 0px;
     width: auto;
-    padding: 3px 35px;
+    padding-right: 35px;
+    padding-left: 35px;
   }
 
   &.whiteButton {
     background: linear-gradient(white, @lightGray);
-    border: 2px solid white;
+    border: 1px solid white;
     box-shadow: @buttonShadow;
     cursor: pointer;
-    font-weight: normal;
-    font-style: normal;
     color: @darkGray;
+
+    &:hover {
+      border: 1px solid @gray;
+    }
   }
 
   &.inlineButton {
     display: inline;
   }
 
-  &.secondaryButton {
-    background-color: #eee;
-  }
-
-  &.secondaryAltButton {
-    background-color: #fff;
-    color: @braveOrange;
-    border: 1px solid #ccc;
-  }
-
   &.subtleButton {
     background-color: #ccc;
     font-size: 14px;
   }
+}
 
-  &.tactileButton {
-    display: block;
-    min-width: 80px;
-    height: 2em;
-    line-height: 2em;
-    margin: 0.5em 0;
-    vertical-align: middle;
-    text-align: center;
-    border: solid 1px @braveMediumOrange;
-    background: @braveMediumOrange;
-    border-radius: @borderRadius;
-    color: white;
-    box-shadow: @lightBoxShadow;
-    padding-left: 1em;
-    font-size: 0.9em;
-    font-weight: 400;
-    cursor: pointer;
-    outline: none;
-    text-align: left;
+// for about:preferences
```

Comment:
The buttons under this section should be removed if necessary. especially from L171 to L176.